code401challenges/README.MD

```diff
@@ -126,4 +126,21 @@ The Stack instances have only push, pop, and peek methods. You should use your o
 ## Solution
 ![Image description](assets/QueueWithStacks.JPG)
 
+# Code Challenge 12 : First-in, First out Animal Shelter.
+
+## Challenge
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+
+## Approach & Efficiency
```

Comment:
Are these the time and space complexity for the `enqueue` or `dequeue` method? Or both?

Reply:
No. The O(n) was for my string builder method which iterates through the queue. I went ahead and updated the space and time complexity to be more explicit